test/config/wfe.json

```diff
@@ -39,7 +39,6 @@
       }
     },
     "features": {
-      "StripDefaultSchemePort": true
```

Comment:
In general, we try to have test/config/* reflect what is _actually_ in prod/staging configs. So it might be vaguely better to leave this here (and in wfe2.json) until someone comes along and removes the feature flag entirely. But it's also not necessary; up to you.

Reply:
I added it back to config for future cleanup, but left it removed from config-next.